### PR TITLE
add tests and cleanup and file flag

### PR DESCRIPTION
**Pull request by gbooth27**
adds testing and the ability to change the name of the output file

### CODE REVIEW

File: cmd/secret/secret.go [hunk 1]
```diff
@@ -47,6 +55,9 @@ var PopulateSecretsCMD = &cobra.Command{
 			logrus.Fatal(errors.Wrap(err, "while getting region flag value"))
 		}
 		persist, err := cmd.Flags().GetBool("persist")
+		if err != nil {
```
Comment (betsy-fu) [inline cmd/secret/secret.go:58]: I think you want an equivalent check in `non-secret.go`

Comment (betsy-fu) [reply cmd/secret/secret.go:58]: Also curious why you opted not to use `viper` like in the `outline2.0/outlinex`?

Comment (gbooth27) [reply cmd/secret/secret.go:58]: what exactly would adding it in here get us? happy to do it but wanted to reduce the complexity

Comment (betsy-fu) [reply cmd/secret/secret.go:58]: just consistency across repos and less `err != nil` checks, but I don't think it's necessary to change now if you have it working already

**Review by betsy-fu — APPROVED** [review]

File: cmd/secret/secret.go (hunk 1 above)
Comment (gbooth27) [reply cmd/secret/secret.go:58]: ah ok! yeah from looking at viper it is for when more config vars are needed (which is the case for outlinex) but here we dont use nearly as many

**Pull request merged**